tests/test_routes.py

```diff
@@ -11,7 +11,6 @@ def app():
     app.config['CONTACT_EMAIL'] = 'mock@example.com'
```

Comment:
Can we hardcode these for now as they're not going to vary by environment?

Reply:
Hardcode them into the values we actually hope to use (i.e. use the DHLUC's actual department name and url info)?